app/shared/video-player/src/commonMain/kotlin/ui/progress/PlayerControllerBar.kt

```diff
@@ -114,6 +135,107 @@ object PlayerControllerDefaults {
+            controllerState.setRequestAlwaysOn(audioIconRequester, isHovered)
+        }
+        Box(
+            modifier = modifier.hoverable(hoverInteraction)
+                .clip(CircleShape)
+                .ifThen(isHovered) {
+                    background(aniColorScheme().surface.copy(alpha = .1f))
+                },
+            contentAlignment = Alignment.BottomCenter,
+        ) {
+            val iconButton = @Composable {
+                IconButton(
+                    onClick = onClick,
+                ) {
+                    when {
+                        isMute -> {
+                            Icon(Icons.AutoMirrored.Rounded.VolumeOff, contentDescription = "静音")
+                        }
+
+                        volume < 0.33f -> {
+                            Icon(Icons.AutoMirrored.Rounded.VolumeMute, contentDescription = "音量")
+                        }
+
+                        volume < 0.66f -> {
+                            Icon(Icons.AutoMirrored.Rounded.VolumeDown, contentDescription = "音量")
+                        }
+
+                        else -> {
+                            Icon(Icons.AutoMirrored.Rounded.VolumeUp, contentDescription = "音量")
+                        }
+                    }
+                }
+            }
+
+            iconButton()
+
+            Popup(
+                alignment = Alignment.BottomCenter,
+            ) {
+                Box(
+                    modifier = Modifier
+                        .hoverable(hoverInteraction)
+                        .clip(shape = CircleShape)
+                        .background(aniColorScheme().surface),
+                ) {
+                    Column(
+                        horizontalAlignment = Alignment.CenterHorizontally,
+                    ) {
+                        AnimatedVisibility(
+                            visible = isHovered && !isMute,
+                            enter = fadeIn() + expandVertically(),
+                            exit = fadeOut() + shrinkVertically(),
+                        ) {
+                            Column(
+                                horizontalAlignment = Alignment.CenterHorizontally,
+                            ) {
+                                Text(
+                                    text = volume.times(100).roundToInt().toString(),
+                                    modifier = modifier.padding(8.dp),
+                                )
+                                VerticalSlider(
+                                    value = volume,
+                                    onValueChange = onchange,
+                                    interactionSource = interactionSource,
+                                    modifier = Modifier.width(96.dp),
+                                    thumb = {},
+                                    colors = SliderDefaults.colors(
+                                        inactiveTrackColor = aniDarkColorTheme().onSurface,
```

Comment:
```suggestion
                                        inactiveTrackColor = MaterialTheme.colorScheme.onSurface,
```

让外部 (VideoScaffold) 去指定 dark